### ISSUE TITLE
errata in chapter 4?

### ISSUE THREAD

**Issue by leonardgithub** [PacktPublishing/Building-Machine-Learning-Projects-with-TensorFlow#8]
section: loss function

actual:
In the previous section we saw our approximated [latex]\hat p[/latex] function,

expected:
Is [latex]\hat p[/latex] right?

**Comment by tendence** [comment]
I think here the \hat p is ypred_i.
Do you think so?
